src/conversion/FileProcessor.ts

```diff
@@ -267,8 +267,13 @@ export class FileProcessor {
                 this.processCompilationUnit(builder, this.source.targetFile, libPath, this.source.parseTree);
 
                 try {
+                    let converted: string = String.fromCharCode(...builder.array());
```

Comment:
@mike-lischke I changed this because java.lang.StringBuilder.toString() returns java.lang.String .